projects/vcx.md

```diff
@@ -0,0 +1,97 @@
+
+VCX has been used in industry projects, including mediator & cloud applications previously at Absa, and is a core component behind [Anonyome Lab's DI Edge Agent Mobile SDK](https://anonyome.com/resources/blog/aries-vcx-anonyomes-commitment-to-decentralized-identity/). Instnt is also building using VCX as a key component for their Multipass product.
+
+Coming from the Aries ecosystem, VCX has strived to implement protocols that are interoperable with the other Aries frameworks (Credo, ACA-Py). As part of this mission, VCX is apart of the [OWL test harness](https://github.com/openwallet-foundation/owl-agent-test-harness), where it is tested against other OWF Aries implementations.
+
+Given it's long history, VCX had previously grown in legacy code, however recent efforts have pushed VCX inline with modern DI. These efforts include complete replacement of Indy SDK components with indy-vdr, askar & anoncreds-rs, and moving away from unqualified DIDs/resources with alternatives such as Cheqd Anoncred methods.
+
+VCX aims to continue this trajectory. Some current future plans include:
+* VCX Framework (in development) - A credo-like framework solution for Rust-based Aries/DI consumers.
+* Mobile bindings - Provide Android/iOS mobile bindings to the library, allowing *native* (Kotlin/Swift) mobile applications to utilize VCX functionality.
+* More DID methods - continue growing the suite of DID method options, with modern methods such as did:webvh
+* DIDComm v2 support
+
+# Alignment with the OpenWallet Foundation Mission
+
+VCX supports the OWF's goals by providing an extensive suite of tools to assist in the creation of secure & interopable DI applications.
+* Secure Data Management: Whenever VCX manages data for the consumer, security is always the first consideration. VCX achieves secure data management via utilization of OWF's Askar project for cryptography and encrypted data storage.
+* Interoperability: As mentioned, VCX comes from an Aries ecosystem, where interoperability is at the forefront. VCX holds itself accountable to this by participating in efforts such as the OWF OWL test harness.
+* Open Source Development: VCX has always been community driven, and encourages participation with efforts such as fortnightly community calls and previous participation in the Linux Foundation Mentorship programs.
+* Support for Decentralized Identity: VCX's purpose is to provide tools that make it easier for consumers to develop DI applications. VCX has been proven to succeed in this, with testimonies in the industry (such as from [Anonyome Labs](https://anonyome.com/resources/blog/aries-vcx-anonyomes-commitment-to-decentralized-identity/)).
+
+# Code of Conduct
+
+VCX currently follows the Hyperledger Code of Conduct, which is what the OWF code of conduct is based on.
```

Comment:
Are there plans to refer to the OWF copy just for consistency?

Reply:
I would expect so -- any concern with that @gmulhearn ?

Reply:
yes definitely happy to adopt and refer to OWF copy. Does this section of the vcx.md need to be updated?